app/scripts/modules/dcos/job/general.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container-fluid form-horizontal">
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      ID
+      <!--<help-field key="dcos.serverGroup.stack"></help-field>-->
+    </div>
+    <div class="col-md-7"><input type="text" class="form-control input-sm" name="id" ng-model="$ctrl.general.id"/></div>
+  </div>
+  <!--<div class="form-group row slide-in" ng-if="basicSettings.region.$error.pattern">-->
```

Comment:
Unfortunately it doesn't seem like there is a way to stop people from saving incomplete stages like there is the modal for creating server groups. We can add validation, but displaying an error message on invalid let alone stopping them from saving seems difficult with how they have things setup compared to the server group modal.